### ISSUE TITLE
pa11y finding: Search input lacks name for accessibility APIs

### ISSUE THREAD

**Issue by SeanKilleen** [mmistakes/minimal-mistakes#2180]
A scan using pa11y on my site came back with:

```
 • Error: This searchinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputSearch.Name
   ├── #search
   └── <input type="search" id="search" aria-placeholder="Enter your search term..." class="search-input" tabindex="-1" placeholder="Enter your search term...">
```

I looked it up and it appears adding a title should be sufficient. PR incoming for the language / title attributes

**Comment by JimDrury** [comment]
Just raising a PR to allow passing an aria-label in, as you would the placeholder
